rockspec/apisix-master-0.rockspec

```diff
@@ -76,8 +76,8 @@ build = {
     install_variables = {
         INST_PREFIX="$(PREFIX)",
         INST_BINDIR="$(BINDIR)",
-        INST_LIBDIR="$(LIBDIR)",
-        INST_LUADIR="$(LUADIR)",
+        INST_LIBDIR="/usr/local/lib64/lua/5.1",
```

Comment:
Why change this?

Reply:
Ditto：https://github.com/apache/apisix/pull/3335#discussion_r564157996